source/server/spongineer/debugging.rst

```diff
@@ -119,3 +119,102 @@ A ``NoClassDefFoundError`` occurs when the plugin tries to access a class that i
 when the API got adjusted or refactored lately and you're trying to run an older plugin on a newer build of Sponge
 and vice versa. Always try to use the correct version! Either ask the Plugin author which Sponge version he build
 against or try updating/downgrading your SpongeForge or SpongeVanilla to solve this.
+
+Exceptions at Runtime
+=====================
+
+There are two kind of errors that might occur on a modded server, first are plugin internal errors such as a
+``NullPointerException`` a command execution. These exceptions are logged using the default stacktrace mechanism from
+Java. The other kind of error is the error during during Minecraft's ticking of worlds or entities. Sponge handles these
+cases in a special way to provide as much information about those as possible.
+
+Before we look into the details on who caused the exception. Make sure that you read all of the exception, sometimes
```

Comment:
The first sentence is incomplete. Also, who -> what. May I humbly suggest:
"Before we look into the details of what caused the exception, make sure that you read all of the error report. Sometimes the error is explained in the error report itself (or a few lines above or below it); sometimes it is indicated by a warning during startup."

Reply:
No need to be humble, your feedback is always welcome.